client/packages/common/src/utils/BarcodeScannerContext.tsx

```diff
@@ -29,9 +33,12 @@ const { Provider } = BarcodeScannerContext;
 
 const parseBarcodeData = (data: number[] | undefined) => {
   if (!data || data.length < 5) return undefined;
+  const synchronousIdleIndex = data.indexOf(22);
```

Comment:
it's popping up now that I've enabled continuous mode - seems to be a sync idle character, according to the ASCII reference. A little annoying.. without stripping it out, then the parsing fails

Reply:
I think James suggesting a comment

Reply:
and I was thinking that it wasn't needed, partly with the rework of the function and partly because I don't know why it's actually happening. But sure, will add something fwiw